src/main/java/stsjorbsmod/cards/cull/Rebuke.java

```diff
@@ -0,0 +1,57 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.IntangiblePlayerPower;
+import com.megacrit.cardcrawl.vfx.combat.CleaveEffect;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Rebuke extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Rebuke.class);
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.ALL_ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 2;
+    private static final int DAMAGE = 10;
+    private static final int UPGRADE_PLUS_DAMAGE = 3;
+    private static final int SELF_INTANGIBLE = 1;
+
+    public Rebuke() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        baseDamage = DAMAGE;
+        magicNumber = baseMagicNumber = SELF_INTANGIBLE;
+        this.isMultiDamage = true;
+        this.exhaust = true;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster abstractMonster) {
+        addToBot(new ApplyPowerAction(p, p, new IntangiblePlayerPower(p, magicNumber)));
+        int intangibleAmount = magicNumber;
+        if (p.hasPower("IntangiblePlayer")) {
+            intangibleAmount += p.getPower("IntangiblePlayer").amount;
+        }
+        this.baseDamage *= intangibleAmount;
```

Comment:
For cards with dynamic damage like this, we use a pattern of setting baseDamage to 0 and using the CustomJorbsModCard helper `calculateBonusBaseDamage` to cover the dynamic amount; that enables updating the card description dynamically by using `getRawDynamicDescriptionSuffix()` with an `EXTENDED_DESCRIPTION` string for an effect similar to the base game's Blizzard. See SnakeOil.java for a similar example